multisig-helper.py

```diff
@@ -690,18 +707,27 @@ def generate_multisig_account(name, ledger, google_api_token, config_path):
     body = {
         'values': [[json_result["address"]]]
     }
-    sheet.values().update(
-        spreadsheetId=config["spreadsheet_id"], range=MULTISIG_ADDRESS_CELL,
-        body=body, valueInputOption="RAW").execute()
 
-    body = {
-        'values': [[" ".join(create_multisig_account_command)]]
-    }
-    sheet.values().update(
-        spreadsheetId=config["spreadsheet_id"], range=GENERATE_MULTISIG_COMMAND_CELL,
-        body=body, valueInputOption="RAW").execute()
+    result = sheet.values().get(spreadsheetId=config["spreadsheet_id"],
+                                range=MULTISIG_ADDRESS_CELL).execute()
 
-    print("Done")
+    if not result["values"][0][0]:
```

Comment:
The script fails on this line when a multisig address cell in Google Sheets is empty:
`KeyError: 'values'`